### ISSUE TITLE
Tests fail

### ISSUE THREAD

**Issue by sbmueller** [sbmueller/gr-openssl#1]
`ctest -V` yields

``` Bash
2: Test command: /usr/bin/sh "/home/senpo/src/gr-openssl/build/python/qa_sym_enc_tagged_bb_test.sh"
2: Test timeout computed to be: 9.99988e+06
2: Traceback (most recent call last):
2:   File "/home/senpo/src/gr-openssl/python/qa_sym_enc_tagged_bb.py", line 24, in <module>
2:     import crypto_swig as crypto
2:   File "/home/senpo/src/gr-openssl/build/swig/crypto_swig.py", line 28, in <module>
2:     _crypto_swig = swig_import_helper()
2:   File "/home/senpo/src/gr-openssl/build/swig/crypto_swig.py", line 24, in swig_import_helper
2:     _mod = imp.load_module('_crypto_swig', fp, pathname, description)
2: ImportError: /home/senpo/src/gr-openssl/build/lib/libgnuradio-crypto.so: undefined symbol: PKCS5_PBKDF2_HMAC
2/7 Test #2: qa_sym_enc_tagged_bb .............***Failed    0.54 sec
```

**Comment by sbmueller** [comment]
There is still a linking problem after building. We need to further investigate CMake config.

**Comment by krisabo** [comment]
There is no problem on my Ubuntu installation, but the tests on my Fedora installation fails.

**Comment by sbmueller** [comment]
When correctly configured, it should work on any distro.

**Comment by sbmueller** [comment]
@risteon We need expert knowledge here 😜

**Comment by sbmueller** [comment]
Resolved in #3
